scss/bitstyles/tools/_flex.scss

```diff
@@ -13,6 +13,14 @@
   align-items: stretch;
 }
 
+@mixin flex--wrap {
+  flex-wrap: wrap;
+}
+
 @mixin flex__primary {
-  flex: 1 0 auto;
+  flex-grow: 1;
```

Comment:
This might be a breaking change. Do we have a changelog in case we publish new versions?
Or maybe increase the version number in package.json?

Reply:
We don’t have a changelog so far — any advice how best to create one? When this batch of updates is on master, I’ll create a new release with a bumped version number, and if we have it by then, a changelog :)